docs/manual-CN/3.build-develop-and-administration/3.configurations/2.configs-syntax.md

```diff
@@ -116,7 +43,7 @@ nebula> GET CONFIGS storage:v;
 UPDATE CONFIGS [graph|storage :] <var> = <value>
 ```
 
-> 更新的变量将持久化在 meta-service 中。
+> 更新的变量将持久化在 `meta service` 中。
```

Comment:
why deleted the introduction? I think we only delete the `gflag` section.

Reply:
I think the introduction is duplicate.